UCLA Radio/UCLA Radio/menu/Schedule/ScheduleViewController.swift

```diff
@@ -3,23 +3,43 @@
 //  UCLA Radio
 //
 //  Created by Christopher Laganiere on 6/3/16.
-//  Copyright © 2016 UCLA Student Media. All rights reserved.
+//  Copyright © 2016 ChrisLaganiere. All rights reserved.
 //
 
 import Foundation
 import UIKit
 
-class ScheduleViewController: ScheduleBaseViewController {
+private let reuseIdentifier = "ScheduleCell"
+private let headerReuseIdentifier = "ScheduleHeader"
```

Comment:
use `fileprivate` here

just `private` limits to scope - that's fine if it's in a class but this is just at file level